grpc node 0.10.0 fails to build as newer nan package is released

There nan 1.9.0 released today, see  https://github.com/nodejs/nan/blob/master/CHANGELOG.md.

It looks like grpc node package fails to build now.  It pulls the 1.9.0 as the dep in package.json says ""nan": "^1.5.0","

The errors are:

> grpc@0.10.0 install /usr/lib/node_modules/grpc
> node-gyp rebuild
gyp WARN EACCES user "undefined" does not have permission to access the dev dir "/root/.node-gyp/0.10.40"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/lib/node_modules/grpc/.node-gyp"
make: Entering directory '/usr/lib/node_modules/grpc/build'
  CXX(target) Release/obj.target/grpc/ext/byte_buffer.o
  CXX(target) Release/obj.target/grpc/ext/call.o
../ext/call.cc: In static member function 'static v8::Handle<v8::Value> grpc::node::Call::StartBatch(const v8::Arguments&)':
../ext/call.cc:592:52: error: 'v8::Local<v8::Value> NanThrowError(const char*, int)' is deprecated (declared at ../node_modules/nan/nan.h:1290) [-Werror=d
eprecated-declarations]
     return NanThrowError("startBatch failed", error);
                                                    ^
../ext/call.cc: In static member function 'static v8::Handle<v8::Value> grpc::node::Call::Cancel(const v8::Arguments&)':
../ext/call.cc:606:48: error: 'v8::Local<v8::Value> NanThrowError(const char*, int)' is deprecated (declared at ../node_modules/nan/nan.h:1290) [-Werror=d
eprecated-declarations]
     return NanThrowError("cancel failed", error);
                                                ^
cc1plus: all warnings being treated as errors
grpc.target.mk:106: recipe for target 'Release/obj.target/grpc/ext/call.o' failed
make: *** [Release/obj.target/grpc/ext/call.o] Error 1
make: Leaving directory '/usr/lib/node_modules/grpc/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
gyp ERR! System Linux 3.16.0-0.bpo.4-amd64
gyp ERR! command "node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/lib/node_modules/grpc
gyp ERR! node -v v0.10.40
gyp ERR! node-gyp -v v1.0.1
gyp ERR! not ok

--- a/src/node/binding.gyp
+++ b/src/node/binding.gyp
@@ -11,7 +11,8 @@
         '-pedantic',
         '-g',
         '-zdefs',
-        '-Werror'
+        '-Werror',
+        '-Wno-error=deprecated-declarations'
       ],
       'ldflags': [
         '-g'

